app/assets/stylesheets/helpers/_contents-list.scss

```diff
@@ -0,0 +1,91 @@
+.contents-list {
+  margin-bottom: $gutter-two-thirds;
+
+  // Always render the contents list above a
+  // back to contents link
+  position: relative;
+  z-index: 1;
+  background: $white;
+  box-shadow: 0 20px 15px -10px $white;
+
+  li {
+    padding-top: $gutter / 4;
+    line-height: 1.3;
+    list-style-type: none;
+  }
+
+  // Remove underlines from lists of links
+  // to improve legibility
+  a {
+    text-decoration: none;
+
+    &:hover,
+    &:focus,
+    &:active {
+      text-decoration: underline;
+    }
+  }
+
+  &.contents-list-nested,
+  &.contents-list-dashed {
+    li {
+      margin-left: $gutter-half + 5;
+      padding-right: $gutter-half + 5;
+      list-style-type: disc;
+
+      @supports (content: "—") {
```

Comment:
I had reason to keep the `@supports` when I switched away from the hack in April – however I haven't been clear in the commit message and I can't remember why. https://github.com/alphagov/government-frontend/commit/827a4af6a90580846f27edf64591f0754c9cae7f

This code isn't really being changed by this PR, just moved. Might be worth assessing as a separate issue.